### PR TITLE
part 6 final

### PR DESCRIPTION
**Pull request by PBueltel**
I didn't realize that I was missing some things from the 2nd fan page assignment. Now that I think about it, I think that's what I was working on when I got the message from you to work on the validation lab, & I guess I forgot that I still had work to do on it.
so I updated this time with both the 2nd & 3rd assignments (plus I added a few extra things that I hope enhances user experiences!)

### CODE REVIEW

**Comment by PBueltel** [comment]
part 4.

I had a question that I emailed you, but it wasn't regarding any of the instructions in this part.

I was also wondering how brackets is telling me I should enter fallback colors for the part I used an hsla color.. I guess it means like how we have fallback font , it wants me to have an hex & rgb as a fallback but the book doesn't say how to do that in Chap 6.

I'll email that question to you too if I remember, thanks

**Comment by PBueltel** [comment]
part 5

**Comment by PBueltel** [comment]
part 6

**Comment by PBueltel** [comment]
forgot couple things, now should be complete part 6

**Comment by PBueltel** [comment]
fixed an error with the pacific resort files, hope this works!!

**Comment by PBueltel** [comment]
ok, i'm trying this way & shold work, I'll email you the prob I was having if you cant tell already from your end

**Comment by PBueltel** [comment]
oh man I think I finally fixed the prob!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <main>
+            <h2>Art of Design</h2>
+            <div class="pindent">
+                <p>Web Design is now one of my favorite things to do and a big reason is because there's virtually no limits to what can be created. Web Design is only limited by technology which, as you know, is only advancing every single day. The technology used today for Web Design already gives us the tools and power to create almost anything we can imagine.</p>
+                <p class="topp">I've only just started learning Web Design and I've created this Website to demonstrate some of what I've learned thus far and to demonstrate why I like Web Design. It's only been about 35 days, but I've already learned a lot of cool things.
+                    <br>Please click the links above to checkout some of the projects I've been working on as well as the links to view the color palate I used for this website and the picture gallery I added for some fun and entertainment.</p>
+            </div>
+            <div class="p">
+                <img id="floatleft" src="images/13.jpg" alt="waterfall" height="350" width="152">
+                <h3>My First Page</h3>
```
Comment (mbMosman) [inline index.html]: You might put each of the sub-sections into HTML5 article or section tags and use these to separate them a little more vertically.  That would help give more open space on the page to aim for that 60% open 30% content 10% action/special ideal

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    <br> practice makes perfect!</p><img id="floatright" src="images/11111.jpg" alt="waterfall" height="152" width="225">
+                <h3>Picture Gallery</h3>
+                <p>Creating a picture gallary is just one more thing about Web Design that I really enjoy. Creating a picture gallery is a great way to not only share pictures with family and friends, but it is also an area that can add some entertainment for them and anyone else who may be visiting the site. </p>
+                <p>One of the best things about Web Design is the moment a site you designed goes live and the site becomes available for anybody in the World to see!
+                </p>
+            </div>
+            <br class="clearboth">
+        </main>
+        <footer>
+            <span class="bold"><a href="http://paletton.com/palette.php?uid=b2I0W3M0kllllaFw0g0qFqFg0w0aF" target="_blank">My&nbsp;Paletton.com&nbsp;Color&nbsp;Palate</a></span>
```
Comment (mbMosman) [inline index.html]: The color palette and the use on this page is awesome, it's very relaxing.  Open the spacing up a bit more and it'll be an awesome "show-off" page.

File: music/style.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    color: #FFFF00;
+    font-family: Georgia, Palatino, "Times New Roman", serif;
+    height: 60px;
+    text-transform: uppercase;
+}
+
+main {
+    border: 2px;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (mbMosman) [inline music/style.css]: These aren't needed here as they are on the wrapper style which is the one that actually sets the width.  The auto margins only work where there is a width set, so doing both on the same style is more typical.

**Pull request closed without merging**